test/e2e/clustertask/start_test.go

```diff
@@ -47,47 +51,55 @@ func TestClusterTaskInteractiveStartE2E(t *testing.T) {
 	kubectl := cli.NewKubectl(namespace)
 	tkn, err := cli.NewTknRunner(namespace)
 	assert.NilError(t, err)
+	// Set environment variable NO_CLUSTERTASK_FOUND_TEST to any value to skip "No ClusterTasks found" test
+	if os.Getenv("NO_CLUSTERTASK_FOUND_TEST") == "" {
+		t.Run("Get list of ClusterTasks when none present", func(t *testing.T) {
+			res := tkn.Run("clustertask", "list")
+			expected := "No ClusterTasks found\n"
+			res.Assert(t, icmd.Expected{
+				ExitCode: 0,
+				Err:      icmd.None,
+				Out:      expected,
+			})
 
-	t.Run("Get list of ClusterTasks when none present", func(t *testing.T) {
-		res := tkn.Run("clustertask", "list")
-		expected := "No ClusterTasks found\n"
-		res.Assert(t, icmd.Expected{
-			ExitCode: 0,
-			Err:      icmd.None,
-			Out:      expected,
 		})
-	})
-
+	}
 	t.Logf("Creating clustertask read-clustertask")
-	kubectl.MustSucceed(t, "create", "-f", helper.GetResourcePath("read-file-clustertask.yaml"))
+	res := kubectl.MustSucceed(t, "create", "-f", helper.GetResourcePath("read-file-clustertask.yaml"))
+	regex := regexp.MustCompile(`read-clustertask-[a-z0-9]+`)
+	clusterTaskName := regex.FindString(res.Stdout())
 
 	t.Logf("Creating git pipeline resource in namespace: %s", namespace)
 	kubectl.MustSucceed(t, "create", "-f", helper.GetResourcePath("git-resource.yaml"))
 
 	t.Run("Get list of ClusterTasks", func(t *testing.T) {
 		res := tkn.Run("clustertask", "list")
-		expected := builder.ListAllClusterTasksOutput(t, c, map[int]interface{}{
-			0: &builder.TaskData{
-				Name: "read-clustertask",
-			},
-		})
-		res.Assert(t, icmd.Expected{
-			ExitCode: 0,
-			Err:      icmd.None,
-			Out:      expected,
-		})
+		if os.Getenv("NO_CLUSTERTASK_FOUND_TEST") == "" {
```

Comment:
We don't need to skip for this test

Reply:
If there are no clustertasks when we run the test then expected value of `tkn ct ls` is  `read-clustertask`. But when we have other clustertasks when we run this test then the expected value of `tkn ct ls` is `read-clustertask` along with other task. So I am running test depending on environment variable